miso-web/src/spring-test/java/uk/ac/bbsrc/tgac/miso/webapp/springtest/HotRestControllerST.java

```diff
@@ -0,0 +1,118 @@
+package uk.ac.bbsrc.tgac.miso.webapp.springtest;
+
+import org.junit.Test;
+
+import org.springframework.web.servlet.*;
+
+import static org.springframework.test.web.servlet.setup.MockMvcBuilders.*;
+
+import javax.ws.rs.core.MediaType;
+
+import org.checkerframework.checker.units.qual.Temperature;
+import org.junit.Before;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.*;
+import org.springframework.test.web.servlet.ResultActions;
+import com.jayway.jsonpath.JsonPath;
+
+import static org.hamcrest.Matchers.*;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+
+import org.springframework.test.web.servlet.MvcResult;
+import uk.ac.bbsrc.tgac.miso.dto.Dtos;
+
+import org.springframework.scheduling.annotation.EnableAsync;
+import org.springframework.web.servlet.View;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import org.springframework.security.test.context.support.WithMockUser;
+import uk.ac.bbsrc.tgac.miso.core.data.type.StatusType;
+import static org.junit.Assert.*;
+import java.util.Collections;
+import uk.ac.bbsrc.tgac.miso.webapp.controller.rest.HotRestController.SpreadsheetDataDto;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.util.List;
+import java.util.Arrays;
+import java.util.ArrayList;
+
+import org.springframework.test.web.servlet.MockMvc;
+import java.util.Date;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import org.springframework.mock.web.MockHttpServletResponse;
+
+
+public class HotRestControllerST extends AbstractST {
+
+  private static final String CONTROLLER_BASE = "/rest/hot";
+
+  @Test
+  public void testDownloadSpreadsheet() throws Exception {
+    SpreadsheetDataDto sheet = new SpreadsheetDataDto();
+
+    ArrayList<String> headers = new ArrayList<String>(Arrays.asList("1", "2", "3", "4", "5"));
+    ArrayList<String> firstrow = new ArrayList<String>(Arrays.asList("one", "two", "three", "four", "five"));
+    ArrayList<String> secondrow = new ArrayList<String>(Arrays.asList("un", "deux", "trois", "quatre", "cinq"));
+    List<List<String>> rows = new ArrayList<List<String>>(Arrays.asList(firstrow, secondrow));
```

Comment:
Arrays.asList already returns an ArrayList, don't need to wrap the call in the constructor unless you need the List to be modifiable, which it doesn't look like you do